scripts/collins.sh

```diff
@@ -32,7 +32,18 @@ JMX_OPTS="-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=333
 DEBUG_OPTS="-XX:ErrorFile=${LOG_HOME}/$APP_NAME/java_error%p.log -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/tmp/collinsDump.hprof"
 
 # Check for config overrides
-[ -f /etc/sysconfig/collins ] && . /etc/sysconfig/collins
+DISTRO=$(lsb_release -d | awk '{print $2}')
```

Comment:
this will explode if you dont have lsb-core installed. Most systems have, but cant depend on this on some RHELs. Can you fall back to processing /etc/system-release instead? /etc/os-release would be dope, but its systemd-dependent.